osu.Game.Tests/Visual/TestCaseEditorComposeTimeline.cs

```diff
@@ -30,7 +31,7 @@ public class TestCaseEditorComposeTimeline : EditorClockTestCase
         };
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(AudioManager audioManager)
```

Comment:
not necessary right?

Reply:
Oops, must have left that in by mistake.